packages/block-library/src/avatar/index.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Server-side rendering of the `core/avatar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/avatar` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Return the avatar.
+ */
+function render_block_core_avatar( $attributes, $content, $block ) {
+	$size               = isset( $attributes['size'] ) ? $attributes['size'] : 96;
+	$wrapper_attributes = get_block_wrapper_attributes();
+
+	$image_styles = array();
+
+	// Add border width styles.
+	$has_border_width = ! empty( $attributes['style']['border']['width'] );
+
+	if ( $has_border_width ) {
+		$border_width   = $attributes['style']['border']['width'];
+		$image_styles[] = sprintf( 'border-width: %s;', esc_attr( $border_width ) );
+	}
+
+	// Add border radius styles.
+	$has_border_radius = ! empty( $attributes['style']['border']['radius'] );
+
+	if ( $has_border_radius ) {
+		$border_radius = $attributes['style']['border']['radius'];
+
+		if ( is_array( $border_radius ) ) {
+			// Apply styles for individual corner border radii.
+			foreach ( $border_radius as $key => $value ) {
+				if ( null !== $value ) {
+					// Convert camelCase key to kebab-case.
+					$name = strtolower( preg_replace( '/(?<!^)[A-Z]/', '-$0', $key ) );
```

Comment:
Can we use the [`_wp_to_kebab_case()` util](https://developer.wordpress.org/reference/functions/_wp_to_kebab_case/) here?